packages/base-zone/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "exports": {
     ".": "./src/index.js",
+    "./zone-helpers.js": "./zone-helpers.js",
```

Comment:
Reviewers, opinions please on how to export things like `prepareRevocableKit`, which are not part of the polymorphic `Zone` abstraction, nor an implementation of it, but rather the first of what's likely to be a bunch of little helper utilities that are parameterized over zones.

Reply:
This approach seems fine.  I've sometimes put these helpers in `./tools/`, but I think some of the team disapproves of that convention.